tests/foreman/longrun/test_inc_updates.py

```diff
@@ -36,109 +35,104 @@
 
 
 @pytest.fixture(scope='module')
-def module_lce_library(module_entitlement_manifest_org):
+def module_lce_library(module_sca_manifest_org):
     """Returns the Library lifecycle environment from chosen organization"""
     return (
         entities.LifecycleEnvironment()
```

Comment:
```suggestion
        module_target_sat.api.LifecycleEnvironment()
```

Reply:
fyi.. replacement of `entities` with `module_target_sat.api` will be taken care by Omkar. So I am not changing non affected part of test module.